src/modeler/AutoRest.Swagger.Tests/SwaggerModelerValidationTests.cs

```diff
@@ -594,6 +594,13 @@ public void PutResponseResourceValidationTest()
         }
 
         [Fact]
+        public void LROStatusCodesValidationTest()
```

Comment:
could run this rule on the repo and see how much it flags? It'd be good to know if the cases are real or if it starts flagging false positives.